### PR TITLE
Update base code block styles to match code snippet pattern

### LINKED ISSUE #3463: Update base code block styles

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

Updated the base code style, and the code-numbered pattern, to match the one seen in the [new code snippet design](https://app.zeplin.io/project/5fa270d8eee4dbb267e579f1/screen/5fc616c95320fdb32885432e)

Fixes #3463 & #3465

## QA

- Open [code block example](https://vanilla-framework-3483.demos.haus/docs/examples/base/code-block)
- Check that the code block matches the design, and looks similar to the blocks in the [code snippet example](/docs/examples/standalone/patterns/code-snippet/code-snippet)
- Open [code numbered example](https://vanilla-framework-3483.demos.haus/docs/examples/patterns/code-numbered)
- Check that the code numbered block matches the design

### CODE REVIEW

File: scss/_base_code.scss [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @import 'settings';
 
+$color-pre-bg: rgba($color-x-dark, 0.03);
```
Comment (lyubomir-popov) [inline scss/_base_code.scss:3]: isthis a new variable? or is this what is used in the pr merged yesterday?

Comment (sowasred2012) [reply scss/_base_code.scss:3]: Yeah, this is what was used there - I moved it here and changed the name for this PR, previously it was more obviously specific to the code snippet.

**Comment by lyubomir-popov** [comment]
Great

**Review by lyubomir-popov — APPROVED** [review]
LGTM

**Pull request merged**